### PR TITLE
site refresh for 2016.46

### PR DESCRIPTION
**Pull request by mystiberry-okta**
@jpf or @johngronberg-okta do you want to review before I update the site?

### CODE REVIEW

File: feed.xml [hunk 1]
```diff
@@ -79,7 +79,7 @@
 
 &lt;p&gt;&lt;strong&gt;Happy Okta’ing!&lt;/strong&gt;&lt;/p&gt;
 </description>
-        <pubDate>Tue, 22 Mar 2016 00:00:00 -0700</pubDate>
+        <pubDate>Tue, 22 Mar 2016 00:00:00 +0000</pubDate>
```
Comment (johngronberg-okta) [inline feed.xml]: Why is this timezone changing?

**Comment by johngronberg-okta** [comment]
Other than the timezone question, LGTM. We HAVE to fix those newlines that get inserted though...

File: feed.xml (hunk 1 above)
Comment (mystiberry-okta) [inline feed.xml]: Safe to publish? The release is out on preview already.

Comment (mystiberry-okta) [inline feed.xml]: JF says go for it.

**Pull request merged**